static/app/views/onboarding/components/stepHeading.tsx

```diff
@@ -22,7 +22,7 @@ const StepHeading = styled(motion.h2)<{step: number}>`
     background-color: ${p => p.theme.yellow300};
     border-radius: 50%;
     color: ${p => p.theme.textColor};
-    font-size: 1.5rem;
+    font-size: 1rem;
```

Comment:
Current font-size is 15px. This is not standard, we should use 16px (`1rem` in the new scale) instead.

With new size:
<img width="391" alt="Screen Shot 2021-12-16 at 11 41 21 AM" src="https://user-images.githubusercontent.com/44172267/146445352-b83a4256-b20d-46d0-aab7-452841ba8c47.png">